### DISCUSSION TITLE
Readme

### DISCUSSION THREAD

**Discussion by Tiyea** [community/community#35418] [Other Feature Feedback, Questions, & Ideas]
Im want to create a readme for my account
But the Initialize this repository with a readme checkbox is not exist:

![image](https://user-images.githubusercontent.com/106560146/194527578-2510f118-965b-4a38-9d2d-518dc0ecdca4.png)

**Comment by DSDmark** [comment]
## if you want to create readem for git profile just do some steps

- Click on + icon at top right of the page and select New Repository. 

- A Create a new repository page opens. In the Repository name field,
enter the username of your GitHub account. After entering the username, GitHub displays a message describing that you’re about to create a GitHub special repository. 

like this :-

![1638950865github-repo-name](https://user-images.githubusercontent.com/85744816/194529903-868aa09e-9acb-4450-9e87-a6ebbe92888d.jpg)

## about the error
> the repository tiyea has been retired and connot be reused

just create a repo first and then after the repo is created then try to add file with the name of **Readme.md**
 I think it should work

 ## if it is not work then here some tips you can try :-  

- First change your username to another name, then set the repository name you want, and finally change the username back, and you're done.

Here is the same discussion about the error "The repository [name] has been retired and cannot be reused"
 The repository [name] has been retired and cannot be reused:-
- https://github.com/orgs/community/discussions/23480

**Reply by rodrigo-herrerab** [reply]
USE FILE WITH EXTENSION ".md"

**Reply by homer1960** [reply]
> USE FILE WITH EXTENSION ".md"

Hello

**Comment by jfullstackdev** [comment]
if that is retired, then I doubt you change your username very quickly, it tends to retire repos

check this out

https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/#popular-repository-namespace-retirement

I'm not very sure whether it will be available to you as soon as possible, posting it here might catch the attention of the Staff, other than that you can try Support, but again, there is no assurance

https://support.github.com/contact/account
